mux_test.go

```diff
@@ -405,16 +405,6 @@ func TestPath(t *testing.T) {
 			pathTemplate: `/{type:(?i:daily|mini|variety)}-{date:\d{4,4}-\d{2,2}-\d{2,2}}`,
 			shouldMatch:  true,
 		},
-		{
```

Comment:
@neelance Does failing this test break the intent of your original changes in 0b13a922203ebdbfd236c818efcd5ed46097d690?